config/alerting/vmoperator-rules.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMRule
+metadata:
+  name: vmoperator-rules
+spec:
+  groups:
+    - name: vmoperator
+      rules:
+        - alert: LogErrors
+          expr: sum(rate(operator_log_messages_total{level="error", job=~".*((victoria.*)|vm)-?operator"}[5m])) > 0
+          for: 15m
+          labels:
+            severity: high
```

Comment:
Should we use the same set of `severity` like [other](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/alerts-cluster.yml) VictoriaMetrics rules here, like "critical"?

Reply:
I don't think that it's `critical`, because it doesn't directly affect the deployments and monitoring work, but you are right, we have to use the same severity names. Maybe change to `warning`, wdyt? (+ @f41gh7)

Reply:
I'm fine with warning.

Reply:
done